robotino_bringup/robots/uibk-robotino2/launch/uibk-robotino2.launch

```diff
@@ -80,11 +80,11 @@
 
   <!-- Simple node to move the kinect in the default position on robot's startup
        It also exposes a service /tilt_controller/resetPosition -->
-  <!--node name="robotino_tilt_controller" pkg="robotino_driver" type="robotino_tilt_controller" output="screen">
+  <node name="robotino_tilt_controller" pkg="robotino_driver" type="robotino_tilt_controller" output="screen">
```

Comment:
you are already starting the tilt controller with the start_pan_tilt_controller.launch